docs/source/examples/login.js

```diff
@@ -27,22 +27,28 @@ import {
 } from '@inrupt/solid-client-authn-browser'
 
 // Build a session
-const session = new solidClientAuthn.Session({
-    clientAuthn: solidClientAuthn.getClientAuthnWithDependencies({})},
+const session = new Session({
+    clientAuthn: getClientAuthnWithDependencies({})},
     "mySession"
 );
 
-// Redirect the user to their identity provider...
-await session.login({
-    // The URL of the user's OIDC issuer
-    oidcIssuer: 'https://identityProvider.com', 
-    // The url the system should redirect to after login
-    redirectUrl: 'https://mysite.com/redirect',
-});
+if (!session.sessionInfo.isLoggedIn) {
+  // Redirect the user to their identity provider:
+  // (This moves the user away from the current page.)
+  session.login({
+      // The URL of the user's OIDC issuer
+      oidcIssuer: 'https://inrupt.net',
+      // The url the system should redirect to after login
+      redirectUrl: 'https://mysite.com/redirect',
+  });
+}
 
-onLogin((sessionInfo) => {
-   // Do stuff
+// At https://mysite.com/redirect:
+if (!session.sessionInfo.isLoggedIn) {
+  await session.handleIncomingRedirect(new URL(window.location.href));
+}
 
-});
+// You can now make authenticated requests by passing session.fetch:
+fetchSolidDataset(session.sessionInfo.webId, { fetch: session.fetch });
```

Comment:
Question: Should that be getSolidDataset?

Reply:
Oh - or is this a separate function - not part of any package.

Or, if want to show usage with solid-client, do we need to update the include in this example file?

Reply:
Whoopsie! We've been renaming so often I'm losing track of it myself... And yes, the import should be updated as well. I'll fix that.

Reply:
03da5c0864c528578c74ca8893591cc02de30cfc